firmware/controllers/algo/fuel/injector_model.cpp

```diff
@@ -128,7 +128,7 @@ float InjectorModel::correctShortPulse(float baseDuration) const {
 }
 
 float InjectorModel::correctInjectionPolynomial(float baseDuration) const {
-	if (baseDuration > CONFIG(applyNonlinearBelowPulse)) {
+	if (baseDuration > (CONFIG(applyNonlinearBelowPulse) * 0.001f)) {
```

Comment:
magic constant code smell

is this seconds to MS conversion? let's find a way to make this more readable

Reply:
other way around, but yes